### DISCUSSION TITLE
state_trigger no longer works

### DISCUSSION THREAD

**Discussion by yunxiali** [custom-components/pyscript#668] [Q&A] [answered]
My state_triggers no longer works while the time trigger in the same file works. I haven't changed the file content, and all I can think of is I had to disable/re-enable the extension for a short while because of a lock that was getting stuck physically. Any idea on what could be the issue? I enabled the debug logging but there are no log entries related to state changes etc. I can see time triggers in the logs though.

The triggers are simple stuff like this, nothing advanced
```
@state_trigger("lock.garage_door_lock == 'unlocked'")
@state_trigger("binary_sensor.laundry_opening == 'on'")
def laundry_light_on():
    if light.laundry_light == "off":
        light.turn_on(entity_id="light.laundry_light")
```

**Comment by ALERTua** [comment]
try to isolate the problem one step at a time and come back with your findings on the cause of this behavior. please describe the process of your troubleshooting for me to maybe point out what you didn't try.

**Reply by tleydxdy** [reply]
Could you suggest some things I could try? So far I just tried to remove the extension and re-adding it, and to add log.warning at the top of the files to make sure they are getting loaded by HA.

Edit, oh and I did fix that lock as well.

**Comment by tleydxdy** [comment, marked as answer]
I updated my pyscript with this https://github.com/custom-components/pyscript/pull/651 and that seems to have fixed the issue. No idea if that pr is the fix or it just triggered something to refresh in home assistant though.
